log/view_log.py

```diff
@@ -0,0 +1,10 @@
+from django.http import HttpResponse
+
+def index(request):
+    logs = get_logs()
+    return HttpResponse('<br>'.join(logs))
```

Comment:
https://deploy-preview-61--coala-community.netlify.com/log/ is currently empty

Reply:
hmm... I'll check it.